build.py

```diff
@@ -16,7 +16,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script generates two versions of Blockly's core files:
+# Usage: build.py <0 or more of accessible, core, generators, langfiles>
+# build.py with no parameters builds all files.
+# core builds blockly_compressed, blockly_uncompressed, and blocks_compressed.
+# accessible builds blockly_accessible_compressed,
+#  blockly_accessible_uncompressed, and blocks_compressed.
+# generators builds every <language>_compressed.js.
+# langfiles builds every msg/js/<LANG>.js file.
+
+# This script generates four versions of Blockly's core files:
```

Comment:
Mentioning there are four versions and then only listing two here is kind of confusing. Maybe list all four and discuss "the first two" and the "second two" in the paragraph or point out that these are the first two before listing them?